fabtests/test_configs/ucx/ucx.exclude

```diff
@@ -14,10 +11,36 @@ av_test
 ^fi_dgram
 -e msg
 -e dgram
-
+#
 getinfo
 rdm g00n13s
```

Comment:
Since the negative test is already excluded here, we shouldn't need the -N argument in the CI run